Makefile

```diff
@@ -5,12 +5,15 @@ PREFIX=/usr/local
 BINDIR=$(PREFIX)/bin
 MANDIR=$(PREFIX)/share/man
 GO=go
+TAGGED_RELEASE=$(shell git describe --tags --abbrev=0)
+COMMIT_HASH=$(shell echo -n $(shell git rev-parse --short HEAD) && git diff-index --quiet HEAD -- || echo -n '-dirty')
```

Comment:
Have you tested this on FreeBSD and OpenBSD? Not sure if they support `shell` too.

Reply:
Just tested, seems like no. But there's `!=` operator from BSD make instead which is also supported by the GNU make since 4.0.